HW_5/Ex_26_modul.py

```diff
@@ -0,0 +1,5 @@
+def powAB(a, b):
+    if b == 1: return a
+    if b == 0: return 1
+    if b < 0: return 1 / (a * powAB(a, -b - 1))
+    else: return a * powAB(a, b - 1)
```

Comment:
Такая запись хоть и допустима, но не принята
```suggestion
    if b == 1:
        return a
    if b == 0:
        return 1
    if b < 0:
        return 1 / (a * powAB(a, -b - 1))
    else:
        return a * powAB(a, b - 1)
```